plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -70,7 +73,37 @@ def _coerce_to_cartesian_si(pos):
     return pos_out
 
 
-class Tracker:
+class _SyntheticRadiographySaveRoutine(AbstractSaveRoutine):
```

Comment:
I think this class needs to be updated to match the changes in `AbstractSaveRoutine`.

Reply:
I *think* it should be good now. It works locally, at least.